ctru-rs/src/console.rs

```diff
@@ -1,27 +1,27 @@
+use std::cell::Ref;
 use std::default::Default;
-use std::marker::PhantomData;
 
 use ctru_sys::{consoleClear, consoleInit, consoleSelect, consoleSetWindow, PrintConsole};
 
-use crate::gfx::{Gfx, Screen};
+use crate::gfx::Screen;
 
-pub struct Console<'gfx> {
+pub struct Console<'screen> {
     context: Box<PrintConsole>,
-    _gfx: PhantomData<&'gfx ()>,
+    _screen: Ref<'screen, dyn Screen>,
```

Comment:
Since we're now actually holding on to this data, we should remove the underscore:
```suggestion
    screen: Ref<'screen, dyn Screen>,
```

Reply:
Oh, I didn't think far enough ahead, this causes an unused field warning... Well, we could leave it for now since we might use it in the future (ex. for a wrapped `set_wide_mode`). Or revert it to `_screen` if you want.